test/performance/README.md

```diff
@@ -112,12 +112,89 @@ To generate the sample workload for KubeStellar performance experiments proceed
 
 
 
+### Collect the creation & update timestamps for workload benchmark (OPTIONAL)
 
-5. Clean up the generated workload Kubernetes API objects from your cluster:
+
+To collect the creation and status_update timestamps for the benchmark workload objects proceed as following:  
+
+1. Run the metrics collection script:
+
+   a) cd into the `test/performance` directory from your local copy of the KubeStellar repo, for example:
 
    ```bash
-   $ cd $CL2_DIR
-   $ ./cleanup.sh
+   $ cd .$HOME/kubestellar/test/performance
    ```
 
-### Collect the creation & update timestamps for workload benchmark (OPTIONAL) (TO DO: WORK-IN-PROGRESS)
\ No newline at end of file
+   b) Run the metrics collection script:
+
+   NB: the execution of the following python script requires the installation of the dependencies listed in `requirements.txt`.
+
+   ```bash 
+   $ python3 metrics_collector.py <kubeconfig> <wds-context-name> <its-context-name> <wec-context-name> <number-of-namespaces> <output-directory> 
+   ```
+
+   For example:
+   
+   ```bash 
+   $ python3 metrics_collector.py $HOME/.kube/config wds1 its1 cluster1 2 $HOME/data
+   ```
+
+   Below is a detailed explanation of the input parameters:
+   - `kubeconfig`: path to the kubeconfig file, e.g., `$HOME/.kube/config`
+   - `wds-context-name`: name of the context for the target WDS, e.g., `wds1`
+   - `its-context-name`: name of the context for the target ITS, e.g., `its1`
+   - `wec-context-name`: name of the context for the target WEC, e.g., `cluster1`
+   - `number-of-namespaces`: number of namespaces created in your experiment, e.g., `2`
+   - `output-directory`: path to the directory for the output data files, e.g., `./data/`
+
+   For each namespace, the creation and status_update timestamps will be collected for the workload objects created in `wds1`, manifestwork & workstatus objects created in `its1`, and workload & appliedmanifestworks objects created in `wec1`, for example: let's assume that `num_ns=2`, then the following directories and files will be created: 
+
+   ```bash 
+      $ tree
+      .
```

Comment:
The output goes into the output directory, right? In none of the preceding examples is `.` the output directory. I would stick with `$HOME/data`, that seems to be the more commonly used example in the earlier text.

Reply:
Fixed!